### ISSUE TITLE
Archived messages have NO content!

### ISSUE THREAD

**Issue by aaeneas** [Brummolix/AutoarchiveReloaded#61]
AutoArchiveReloaded (AAR) was recently discovered and the advertised features were quite appealing.  Therefore, decided to give it a try.  First added it to a test system running TB 78.7.1 on Windows 10.  Problem here of course is I do NOT have an IMAP server to test with.  However, I did have an email account that I seldom use and have NOT been bothering to archive the records.  As a result it had around 10 years of messages on the server and I was NOT too concerned about the possibility of either loosing or messing up these messages.  Set the archive date range to 420 days which is close to what I might want to use for real.  Incrementally checked the boxes used to specify which kind of messages are archived and reran AAR after each change.  As best I can tell it all worked marvelously well. Haven't spent a lot of time investigating results but have yet to find a problem.

This lead pretty directly to decision to try it for real.  Went to an active (dependable) system which in this case is TB 68.12.1 on Windows 7.  In this case, the account has been archived over a period of about 15 years. It is what I'd call a private account which I've been able to keep relatively free of SPAM so as email accounts go it has relatively few messages.  Probably less than 10,000 messages per year.  My previous archive was for 2018 so I set the date range to 760 which was expected to affect about 10 days worth of messages in 2019.  That worked well as best I could tell (did NOT thoroughly investigate).  Revised the date range to 700 (i.e., a couple of months more).  In this case the result was disastrous.

While there are several problems the most significant is that messages listed as being in the archive (i.e., look normal in the list pane) have absolutely NO content displayed when the message is opened.  If I view the source the following is displayed:

From - Tue Feb 09 17:25:04 2021
X-Mozilla-Status: 0001
X-Mozilla-Status2: 00000000
X-Mozilla-Keys: 

That's it.  I haven't looked at the source for all of the empty messages but those same 4 lines appear in all of the ones I did look at (maybe a dozen).  The date displayed does correspond with when AAR was run.  In some cases the list view shows duplicates (i.e., exact same information appears in the list entry) and luckily one the those entry opens what looks to be a valid message.

If there is some reasonable course of action I'd be willing to assist with further troubleshooting but needless to say I'm pretty close to giving up on AutoArchiveReloaded.

**Comment by Brummolix** [comment]
That's a pitty. I'm sorry.

Seem to be the same as #14 
As @wsmwk wrote there:
> There are several bug reports which describe losing the message or message body when moving to a local folder https://mzl.la/308Nz8Y

**Comment by aaeneas** [comment]
I've been archiving TB email for more than a decade.  However, the process has been quite tedious.  For most of that time I had no problem selecting a large number of messages (I seem to recall doing it quarterly) which would be thousands of messages and invoking the "Move To" operation from the drop down menu.  While more tedious than I'd like it worked fine for quite a long time.  However, when I went to do my 2018 messages (which looks to have been in mid 2020) I had a similar problem to the one reported herein.  Similar in that message content was lost.  However, in that case the message list contained no relevant data whereas in this case the entries in the message list appear normal.  By changing from "Move To" to "Copy To" I was able to avoid loosing messages when this happens but the problem also occurs for "Copy To".

My interest in AutoArchiveReloaded (AAR) had a lot to do with hoping to find a solution to this problem.  I think what you might be saying is that AAR is using those same basic TB capabilities, which I'm inclined to think only compounds the problem.  Is that correct?

Another observation I'll offer is that AAR doesn't seem to provide any feedback.  In fact, I was unable to figure out whether or NOT it was running.  It would seem to me that it should open a window/tab and display information about what it is doing and especially provide some kind of completion status message when it is done.

**Comment by Brummolix** [comment]
Exactly. AAR does not archive itself. Instead it "only" tells TB to do so. As it uses the TB components it also inherits potential problems of them.
For progress see #55

**Comment by aaeneas** [comment]
It does look like I'm NOT the only one who would like to be able to tell whether or NOT AAR is running.

Given that you undoubtedly know a lot more than me about the underlying processes used for archiving I'm hoping you won't mind my posing a question aimed at finding a way to get around the problem.  It occurs to me that an alternate approach might be better able to avoid the big problem which is loss of messages.

Suppose I were to use AAR but specify a target folder that is on the server rather than local.  I'm guessing that creates quite a different mechanism for placing archived messages in the target folder.  When I drag that remote folder to the local folders it looks like TB does copy rather than move.  In theory then nothing is yet lost unless the mechanism used by the server did it.  Then I could at least inspect the local folder prior to deleting the remote one.  Obviously NOT as automated as one would like but possibly it gets around the data loss problem and takes advantage of some features of AAR (but ability to tell that AAR is running still needed).  How much similarity is there between dragging a folder from remote to local with selecting files from list and using "Copy To"?

What's your opinion of this idea?  I still have most of a years messages on one email account for which I've now created an archive that I could use for test purposes.
